toolkit/scss/shared_placeholders/_visuallyhidden.scss

```diff
@@ -0,0 +1,4 @@
+%visuallyhidden {
+  position: absolute;
+  left: -9999em;
```

Comment:
The reason is to hide the content visually while keeping it available to users who don't use visuals to navigate the page. Here's an article explaining the technique: http://webaim.org/techniques/css/invisiblecontent/

www.gov.uk have styles for all the different use-cases around how to control the display of elements with CSS:

https://github.com/alphagov/govuk_template/blob/86106bfb332ff5c0346b92310a4117daeb2d8c8d/source/assets/stylesheets/_accessibility.scss

From memory, the only reason we position from the left is because standard page-flow is left-to-right but it's not a big problem if we decided to choose a different one instead.

Finally (!) the reason `em`s are used for the unit are because they scale with the text and 9999 `em`s are going to be a lot of pixels (at least 16 x 9999).